app/views/teams/users/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1 class="usa-display">Add New user</h1>
+
+<%= simple_form_for(@user,
+    html: { autocomplete: 'off', role: 'form' },
+    url: team_users_path(@team)) do |form| %>
+  <%= render 'form', form: form %>
+  <div class="margin-top-2">
+    <%= form.button :submit, 'Add', :class => "usa-button" %>
+    <%= link_to 'Back', team_manage_users_path(@team)+'/new', 
```

Comment:
**comment:** based on conversation with @cdelivuk earlier today we'll probably want to remove float-left so that Add comes first.